core/src/main/scala/org/apache/pekko/persistence/jdbc/state/DurableStateQueries.scala

```diff
@@ -54,7 +54,7 @@ import pekko.persistence.jdbc.config.DurableStateTableConfiguration
   }
 
   private[jdbc] def selectFromDbByPersistenceId(persistenceId: Rep[String]) =
-    durableStateTable.filter(_.persistenceId === persistenceId)
+    durableStateTable.filter(_.persistenceId === persistenceId).sortBy(_.revision.desc)
```

Comment:
it seems to me that we should sort this so that the result is based on the revision number - not sure why this is not already the case

Reply:
Ordering is no necessary.

Reply:
removing